spec/controller_services/shopping_lists_controller/destroy_service_spec.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require 'service/no_content_result'
```

Comment:
This was a holdover from when `204 No Content` was an actual possible response from this endpoint.